src/main/java/io/appium/java_client/touch/PositionOffsetOption.java

```diff
@@ -0,0 +1,51 @@
+package io.appium.java_client.touch;
+
+import static java.util.Optional.ofNullable;
+
+import org.openqa.selenium.Point;
+
+import java.util.Map;
+
+public class PositionOffsetOption<T extends PositionOffsetOption<T>> extends ActionOptions<T> {
+    private Point offset = new Point(0, 0);
```

Comment:
like I mentioned before, the default value should not be set to `0,0`, but rather to null. When only element is set the the server code selects appropriate coordinates for touch and this is not always the top left corner of the element.